src/sentry/static/sentry/app/views/projectInstall/createProject.tsx

```diff
@@ -27,6 +28,11 @@ import withOrganization from 'app/utils/withOrganization';
 import withTeams from 'app/utils/withTeams';
 import IssueAlertOptions from 'app/views/projectInstall/issueAlertOptions';
 
+const getCategoryName = (category?: string) => {
+  const matched = categoryList.find(({id}) => id === category);
+  return matched?.id;
```

Comment:
Can you just oneline this?

```tsx
const getCategoryName = (category?: string) =>
  categoryList.find(({id}) => id === category)?.id;
```

Reply:
@EvanPurkhiser I _could_ but I find the whole `)?.id` bit ugly :D

Reply:
LOL, ok I won't stop you since it's ultra subjective

Reply:
I made the change even though I hate it 🙃

JK it's not that bad